app/frontend/components/shared/form/input-form-control.tsx

```diff
@@ -70,6 +71,22 @@ export const FileFormControl = (props: IInputFormControlProps) => {
   return <InputFormControl {...(R.mergeDeepRight({ inputProps: { type: "file" } }, props) as IInputFormControlProps)} />
 }
 
+export const TextAreaFormControl = (props: IInputFormControlProps) => {
+  return (
+    <InputFormControl
+      {...(R.mergeDeepRight(
+        {
+          inputProps: { as: Textarea }, // Use Chakra's Textarea instead of Input
+          validate: {
+            satisfiesLength: (str) => !props.required || (str?.length >= 1 && str?.length < 256) || "Input is invalid", // Use a default error message or use translation as needed
```

Comment:
shouldn't this `satisfiesLength: (str) => (!props.required && !value?.trim() || (str?.length >= 1 && str?.length < 256) || "Input is invalid",` . We still want to validate the length if they enter something even if the input value is not required